supporter-product-data/src/test/scala/conf/ZuoraQuerierConfigSpec.scala

```diff
@@ -0,0 +1,20 @@
+package conf
+
+import com.gu.conf.ZuoraQuerierConfig
+import com.gu.model.Stage.DEV
+import com.gu.test.tags.annotations.IntegrationTest
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+@IntegrationTest
```

Comment:
these new IT tests won't automatically run by the IT test runner, we would need to add them to the relevant config so that it will pick them up.

Reply:
I don't think it's worth running them in the IT test runner to be honest, they are mostly there to help out when developing locally